google-home-community.groovy

```diff
@@ -1327,7 +1335,8 @@ private handleExecuteRequest(request) {
         devices.each { device ->
             def result = results[device.device]
             result.ids = [device.device.id]
-            if (result.status == "SUCCESS") {
+            // Return true regardless of decvice state if immediate reponse requested
+            if (result.status == "SUCCESS" || settings.immediateResponse) {
```

Comment:
`result.status` will only not be `"SUCCESS"` if there was an exception thrown by he command handler.  I don't think this should be checking `settings.immediateResponse`.

Reply:
Ah, I wasn't sure how result.status was set and what the default was. If it is almost always expected to be "SUCCESS" even before completing the command, then as you mentioned, there is no point to check here.